symengine/cwrapper.h

```diff
@@ -5,6 +5,8 @@
 #include <stdlib.h>
 #include <gmp.h>
 
+#include "symengine/matrix.h"
```

Comment:
Don't include C++ headers in this C header. Take a look at `CVecBasic` to see how `vec_basic` is wrapped.